README.md

```diff
@@ -4,3 +4,12 @@
  [![forthebadge](https://forthebadge.com/images/badges/built-with-resentment.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/contains-technical-debt.svg)](https://forthebadge.com) [![forinfinityandbyond](https://user-images.githubusercontent.com/5211576/29499758-4efff304-85e6-11e7-8267-62919c3688a9.gif)](https://www.reddit.com/r/SS13/comments/5oplxp/what_is_the_main_problem_with_byond_as_an_engine/dclbu1a)
 
  This is a fork based off the July-2018 version of ColonialMarines. To see the original, GPL repo, go [here](https://github.com/MrStonedOne/cmhistory)
+
+ # Sound Credits
+[Pack: Footsteps](https://freesound.org/people/soundmary/packs/7395/)  by [soundmary](https://freesound.org/people/soundmary/) used under (CC BY 3.0)[https://creativecommons.org/licenses/by/3.0/] / Multiple files made from original
+
+
+
+ # Sprite Credits
+
```

Comment:
I'd rather we kept the readme clean and just added a CREDITS.txt into the footstep folder where you added the sounds.

Reply:
Done!